_posts/2018-06-11-2018-foreman-survey-analysis.md

```diff
@@ -0,0 +1,406 @@
+---
+layout: post
+title: 2018 Foreman Survey Analysis
+date: 2018-06-11 10:00:00
+author: Greg Sutcliffe
+tags:
+- usage
+- foreman
+- events
+- plugins
+- community
+- survey
+- API
+- virtualization
+- puppet
+- katello
+- pulp
+- content
+- use cases
+---
+
+As with previous years, we ran a Foreman Community Survey in order to give you
+all the opportunity to tell us how we're doing - where it's good, and where
+it's bad. That survey closed a while ago, and I'm here to show you the results.
+
+<!--more-->
+
+Firstly - **thank you** to all those who filled out the survey. We kept the same
+multi-page format since it seems to work, and even without prize incentives, we
+got over 160 responses! You're all legends :)
+
+If you've seen the previous community survey analysis posts, you'll note the
```

Comment:
I found myself wanting to know when the survey officially closed to help understand up to what point in time the survey represents.

Reply:
Good point, I'll add that

Reply:
Sadly Google Forms doesn't store the data. I've put $last-repsonse[timestamp] - $first_response[timestamp] in the text.